sandbox/libs/authn/src/main/java/org/opensearch/authn/AuthenticationTokenHandler.java

```diff
@@ -10,6 +10,11 @@
 import org.apache.shiro.authc.AuthenticationToken;
 import org.apache.shiro.authc.UsernamePasswordToken;
 import org.opensearch.authn.tokens.BasicAuthToken;
+import org.opensearch.authn.tokens.BearerAuthToken;
+import org.apache.shiro.authc.BearerToken;
```

Comment:
We shouldn't be directly referencing Shiro classes in this part of the code, as any AuthenticationManager could be used with the service.  Only clases under the `org.opensearch.authn.internal` should have any references

Reply:
Oh okay; I did not realize this distinction. I will fix this then.